checkov/kubernetes/kubernetes_utils.py

```diff
@@ -18,6 +18,7 @@
 
 K8_POSSIBLE_ENDINGS = {".yaml", ".yml", ".json"}
 DEFAULT_NESTED_RESOURCE_TYPE = "Pod"
+SUPPORTED_POD_CONTAINERS_TYPES = ["Deployment", "DeploymentConfig", "DaemonSet", "Job", "ReplicaSet", "ReplicationController", "StatefulSet"]
```

Comment:
```suggestion
SUPPORTED_POD_CONTAINERS_TYPES = {"Deployment", "DeploymentConfig", "DaemonSet", "Job", "ReplicaSet", "ReplicationController", "StatefulSet"}
```
can you make it a `set`, then the lookups are faster 🙂